netops_demo/golang/src/github.com/v3io/demos/functions/ingest/ingest_test.go

```diff
@@ -39,7 +39,8 @@ func (suite *ingestSuite) TestIngestValid() {
             "longitude": 51.4990232516124,
             "latitute": -0.16853921733979035,
             "company_name": "Wong Ltd",
-            "site_name": "Wong Ltd/0"
+            "site_name": "Wong Ltd/0",
+			"device": "Wong Ltd/0/0"
```

Comment:
Weird indentation going on here. I think your added code is correct (tabs) whereas the previous code was wrong (spaces). Please convert the previous to tabs